<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CItemsMenu</class>
 <widget class="QDialog" name="CItemsMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pbItemsUp">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>90</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbItemsEnter">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>90</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbItemsDown">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>90</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Down</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QTableView" name="tvItems">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>300</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>14</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
